example/README.md

```diff
@@ -0,0 +1,28 @@
+# CSI Mock Driver Example
+
+This folder contains an example manifest of deploying CSIDriver including `csi-driver-node-registrar`, `csi-provisioner`,
+`csi-resizer` and `csi-snapshotter` onto a cluster. For testing purpose, `csi-attcher` is not included. Thus, 
+`"--disable-attach=true"` is set on the csi mock driver.
+
+## Usage
+
+To install the CSI mock driver in your cluster.
+```
+$ kubectl apply -f deploy/csi-mock-driver-rbac.yaml
+$ kubectl apply -f deploy/csi-mock-driver-deployment.yaml
+```
+
+This will deploy a `CSIDriver` called `io.kubernetes.storage.mock`. There will be a deployment `csi-mockplugin` being 
+installed in the default namespace. Correspondingly, you can find a pod with prefix `csi-mockplugin`. In the meantime,
+ a `StorageClass` called `test-csi-mock` will be generated along with a `VolumeSnapshotClass` called `csi-mock-snapclass`.
+
+ Note that if you are using version prior to 1.17, [snapshot-controller](https://github.com/kubernetes-csi/external-snapshotter#usage) will require to be manual installed.
```

Comment:
prior to 1.17, there was no snapshot controller. I think maybe a more accurate note is to say that the k8s distribution is supposed to install the snapshot-controller, but if it doesn't, then you'll have to manually install it separately.

Reply:
Done